app/src/mocks/mock-validator-table.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import { ColumnDef } from '@tanstack/react-table';
+import { ApiData } from 'src/api/types';
+import { NumberedPagination } from 'src/components';
+import { TableOptions } from 'src/store/types';
+import { setValidatorTableOptions, updateValidatorPageNum } from 'src/store';
+import { standardizeNumber, truncateHash } from 'src/utils';
+import { standardizePercentage } from 'src/utils/standardize-percentage';
+import { Link } from 'react-router-dom';
+
+export const getMockCurrentEraValidators = () => ({
+  validators: [
+    {
+      publicKey:
+        '015692c70f62a5227b4af46b90f03b0966725d8101215dfcf395445459e5ba2fad',
+      totalStakeMotes: 115914132215810220,
+      feePercentage: 10,
+      delegatorsCount: 5,
+      selfPercentage: 1.37,
+      percentageOfNetwork: 1.33,
+      rank: 25,
+    },
+  ],
+  status: { validatorsCount: 100, bidsCount: 110, latestEraId: 8888 },
+});
+
+export const getMockNextEraValidators = () => ({
+  validators: [
+    {
+      publicKey:
+        '015692c70f62a5227b4af46b90f03b0966725d8101215dfcf395445459e5ba2fad',
+      totalStakeMotes: 115929495810390480,
+      feePercentage: 10,
+      delegatorsCount: 5,
+      selfPercentage: 1.37,
+      percentageOfNetwork: 1.33,
+      rank: 25,
+    },
+  ],
+  status: { validatorsCount: 100, bidsCount: 110, latestEraId: 8889 },
+});
+
+export const mockValidatorsTableOptions: TableOptions = {
+  pagination: {
+    pageSize: 10,
+    pageNum: 1,
+  },
+  sorting: {
+    sortBy: '',
+    order: 'desc',
+  },
+};
+
+const mockRowCountSelectOptions = [
+  {
+    value: 'string',
+    label: 'string',
+  },
+];
+
+export const getMockValidatorsTableHeader = () => (
```

Comment:
I thought there was more content in the header than just the pagination?

Reply:
The Era toggle buttons are, but in order to render New Era data I found it easier to import then directly into the test rather than mock them.

Reply:
Made the update using `beforeEach`.  However, I had to add new describe block for the Next Era -> Total Stake adjustment